django/db/migrations/operations/special.py

```diff
@@ -97,8 +96,7 @@ def reversible(self):
         return self.reverse_sql is not None
 
     def state_forwards(self, app_label, state):
-        for state_operation in self.state_operations:
-            state_operation.state_forwards(app_label, state)
+        state.run_sql(app_label, self)
```

Comment:
These changes seem wrong and should be revereted. The `state` layer should not have notion of `Operation` we want to avoid adding between bi-directional coupling between both layers.